concent_api/core/message_handlers.py

```diff
@@ -1417,3 +1427,20 @@ def is_message_recieved_in_wrong_state(subtask_id, forbidden_states):
 
 def are_items_unique(items: list):
     return len(items) == len(set(items))
+
+
+def are_eth_addresses_and_singularities_equal_in_all_subtask_results_accepted_list(force_payment: ForcePayment) -> bool:
+    subtask_results_accepted_list = force_payment.subtask_results_accepted_list
+
+    first_key = subtask_results_accepted_list[0].task_to_compute.requestor_ethereum_public_key
+    for subtask_results_accepted in subtask_results_accepted_list:
```

Comment:
Why not performing both checks in single loop ?

Reply:
done in one loop, but i left two `if`. This variables, like `subtask_results_accepted.task_to_compute.requestor_ethereum_public_key` are pretty long and it's hard to read it in one if. I think it is the best way.. unless you have better idea :)

Reply:
You can split `if` into several lines, something like:
```
if (
    first_key != subtask_results_accepted.task_to_compute.requestor_ethereum_public_key or
    first_address != subtask_results_accepted.task_to_compute.requestor_ethereum_address
):
```

Reply:
I have looked at the code now, please combine it into single `if`.

Reply:
ok, done :)